rpc/flipt/flipt.proto

```diff
@@ -34,7 +34,7 @@ message EvaluationResponse {
   map<string, string> request_context = 3;
   bool match = 4;
   string flag_key = 5;
-  string segment_key = 6;
+  string segment_key = 6 [deprecated = true];
```

Comment:
cant make this deprecated if we dont give them another field like `segment_keys` to move to